### PR TITLE
Make manual telemetry exception events comply with :telemetry.span/3

### PR DESCRIPTION
**Pull request by dvic**

### CODE REVIEW

**Review by sorentwo — COMMENTED** [review]
This is great for consistency and entirely backward compatible; very nice! There are a few spots in `README.md`, `telemetry.ex`, and `expected-failures.md` that use `meta.error`. Let's change those to `meta.reason`, then this is ready to go 👍

**Comment by dvic** [comment]
> This is great for consistency and entirely backward compatible; very nice! There are a few spots in `README.md`, `telemetry.ex`, and `expected-failures.md` that use `meta.error`. Let's change those to `meta.reason`, then this is ready to go 👍

Done 👍

File: guides/upgrading/v2.0.md [hunk 1]
```diff
@@ -172,8 +172,8 @@ And update the `attach` calls:
 Finally, replace references to `meta.stack` with `meta.stacktrace`:
 
 ```diff
-- Sentry.capture_exception(meta.error, stacktrace: meta.stack, extra: extra)
-+ Sentry.capture_exception(meta.error, stacktrace: meta.stacktrace, extra: extra)
+- Sentry.capture_exception(meta.reason, stacktrace: meta.stack, extra: extra)
++ Sentry.capture_exception(meta.reason, stacktrace: meta.stacktrace, extra: extra)
```
Comment (sorentwo) [inline guides/upgrading/v2.0.md:176]: This upgrade guide was for 2.0, so `meta.error` is still correct here.

**Review by sorentwo — COMMENTED** [review]
Ok, the updates look good, except for in the upgrade guide. Will you remove that change?

File: guides/upgrading/v2.0.md (hunk 1 above)
Comment (dvic) [reply guides/upgrading/v2.0.md]: Ah okay, I reverted that change.

**Comment by sorentwo** [comment]
Wonderful! 💛

**Pull request merged**